.github/workflows/alt_text.yml

```
@@ -0,0 +1,13 @@
+name: Missing alt text
+on:
+  pull_request:
+    branches: [ "main" ]
+    types: [opened, synchronize]
+
+jobs:
+  check-alt-text:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check markdown for alt text presence 
+        run: bash .github/scripts/alt-text.sh
```

I only noticed this because it failed itself!

```suggestion
        run: bash .github/scripts/alt_text.sh
```

---

whoops lol good catch!